src/main/java/edu/harvard/iq/dataverse/license/License.java

```diff
@@ -73,14 +75,17 @@ public class License {
 
     @Column(nullable = false)
     private boolean isDefault;
+
+    @Column(nullable = true)
```

Comment:
Since we're requiring that all new licenses have sort order, it seems to make sense that we make this column non-nullable. You'd have to also update the migration script to set the sort order = id(or some other value that makes sense) for existing licenses.

Reply:
I made the column not null. I have adjusted the migration script to use 0 as the default value when creating the new column. I think that this is OK, the queries sort first by `sortId`, then by `id`. Setting it to 0 makes it clear that it was not set by admin and the logic of sorting by 'id' is kept, until `sortId` is set to some different value.

Reply:
Setting it to 0 makes sense. Thanks for the update.

I get an error when I run LicensesIT - 

Line 95:
        long licenseId = JsonPath.from(body).<Long>getList("data[*].id").stream().max((x, y) -> Long.compare(x, y)).get();

fails first because the [*] fails to parse 

Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: 
startup failed:
Script1.groovy: 1: unexpected token: ] @ line 1, column 33.
   restAssuredJsonRootObject.data[*].id
                                   ^

1 error

	at edu.harvard.iq.dataverse.api.LicensesIT.testLicenses(LicensesIT.java:95)

Then there's a class cast exception the id is integer not long.

When I changed the line to 

        long licenseId = (long) JsonPath.from(body).<Integer>getList("data.id").stream().max((x, y) -> Integer.compare(x, y)).get();

all of the tests pass for me.

Reply:
Sorry that I missed it, thank you for catching and fixing it! I thought if it was not fixed correctly, it would show up in the maven test, but this one is an integration test, I forgot about that... I did the fix you suggested.